.github/scripts/get-files-sha.sh

```diff
@@ -1,3 +1,25 @@
 #!/bin/bash
 
-echo sha=$(cd $1; git log -1 --pretty=format:"%H" ${@:2}) >> "$GITHUB_OUTPUT"
+repo_path=$1
+path=${@:2}
+
+if [ -n "$GITHUB_EVENT_PULL_REQUEST_HEAD_REF" ]; then
+
+    if ! git rev-parse --verify "$GITHUB_EVENT_PULL_REQUEST_BASE_REF"; then
+        git fetch origin $GITHUB_EVENT_PULL_REQUEST_BASE_REF
+        git branch $GITHUB_EVENT_PULL_REQUEST_BASE_REF FETCH_HEAD
+    fi
+
+    if ! git rev-parse --verify "$GITHUB_EVENT_PULL_REQUEST_HEAD_REF"; then
+        git fetch origin $GITHUB_EVENT_PULL_REQUEST_HEAD_REF
+        git branch $GITHUB_EVENT_PULL_REQUEST_HEAD_REF FETCH_HEAD
+    fi
+
+    sha=$(git log -1 --format="%H" origin/$GITHUB_EVENT_PULL_REQUEST_BASE_REF $path)
+    sha=$sha-$(git log -1 --format="%H" origin/$GITHUB_EVENT_PULL_REQUEST_HEAD_REF $path)
+    echo $sha
+    echo sha=$sha >> "$GITHUB_OUTPUT"
```

Comment:
When `GITHUB_EVENT_PULL_REQUEST_BASE_REF` and  `GITHUB_EVENT_PULL_REQUEST_HEAD_REF` will have shorter name, the lines 18-21 can be just single line command.

Reply:
The suggestion was:
```
echo sha=$(git log -1 --format="%H" origin/$BASE_REF $path)-$sha-$(git log -1 --format="%H" origin/$HEAD_REF $path) >> "$GITHUB_OUTPUT"
```

Reply:
Not needed anymore